create_keepalived_sysext.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download the keepalived from git, checkout the tag (e.g., for v2.2.8), build a static binary and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "The build process requires docker"
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+
+if ! which docker &>/dev/null; then
+  echo Missing docker in path
+  exit 1
+fi
+
+export ARCH=x86-64
+mkdir -p "${SYSEXTNAME}"
+#docker build -t keepalived-build --build-arg  VERSION=$VERSION - <  Dockerfile-keepalived
+docker build -t keepalived-build --build-arg  VERSION=$VERSION - <<-'EOF'
+
+FROM alpine:3.19 as build
+ARG VERSION=not_set_force_fail
+RUN    apk --no-cache add \
```

Comment:
What we could do here is to put all the commands in a temporary "install" script (bash heredoc writing to `${SYSEXTNAME}/install`), and then use `docker run -v "${SYSEXTNAME}":/work /work/install` to run it and prepare the binary in `/work/usr/bin/`.

Reply:
I have updated the build process.

Reply:
Thanks, works well now. I also wanted to test arm64 but am hitting the Docker hub API limit even though I just downloaded one image so far…